app/src/androidTest/java/com/android/sample/ui/map/MapScreenTest.kt

```diff
@@ -55,22 +57,48 @@ class MapScreenTest {
     val stateFlow = MutableStateFlow(uiState)
     Mockito.doReturn(stateFlow).`when`(listActivitiesViewModel).uiState
     `when`(navigationActions.currentRoute()).thenReturn(Screen.MAP)
-    composeTestRule.setContent {
-      MapScreen(navigationActions, locationViewModel, listActivitiesViewModel)
-    }
   }
 
   @Test
   fun testMapScreenIsDisplayed() {
-
+    composeTestRule.setContent {
+      MapScreen(navigationActions, locationViewModel, listActivitiesViewModel)
+    }
     composeTestRule.onNodeWithTag("mapScreen").assertIsDisplayed()
     composeTestRule.onNodeWithTag("bottomNavigationMenu").assertIsDisplayed()
     composeTestRule.onNodeWithTag("centerOnCurrentLocation").assertIsDisplayed()
   }
 
   @Test
   fun testCenterOnCurrentLocationTriggersGetCurrentLocation() {
-
+    composeTestRule.setContent {
```

Comment:
You could use a shared SetContent method here since you setup exactly the same things for both testMapScreenIsDisplayed() and testCenterOnCurrentLocationTriggersGetCurrentLocation()

Reply:
Totally agree with that

Reply:
Understood, thanks!